subiquity/client/controllers/ubuntu_advantage.py

```diff
@@ -31,6 +35,14 @@ class UbuntuAdvantageController(SubiquityTuiController):
 
     async def make_ui(self) -> UbuntuAdvantageView:
         """ Generate the UI, based on the data provided by the model. """
+        path_lsb_release: Optional[str] = None
+        if self.app.opts.dry_run:
+            # In dry-run mode, always pretend to be on LTS
+            path_lsb_release = "examples/lsb-release-focal"
+        if "LTS" not in lsb_release(path_lsb_release)["description"]:
```

Comment:
This PR is getting a little long and gathering some additional items, so I want you to do something in a follow-on PR.
We call lsb_release in six spots, and I'd like them to have consistent answers (even in dry-run mode).
So I'd like to add an argument to lsb_release to tell it to use dry-run mode and then that can trigger the usage of the example data.  Then we can tell all lsb_release usage about dryrun.